### PR TITLE
Add Apollo Service

### PR DESCRIPTION
**Pull request by ecl1pseo**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1 +0,0 @@
-SKIP_PREFLIGHT_CHECK=true
```
[the remark this answers is not in the document]
Comment (ecl1pseo) [reply .env.example:1]: Created .env.example file

File: .env [hunk 2]
```diff
@@ -0,0 +1,3 @@
+SKIP_PREFLIGHT_CHECK=true
+
+REACT_APP_BASE_URI = http://localhost:5000
```
[the remark this answers is not in the document]
Comment (ecl1pseo) [reply .env]: Moved .env file to gitignore

File: src/services/ApolloService/ApolloService.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import ApolloClient from 'apollo-boost';
```
Comment (Alan18081) [inline src/services/ApolloService/ApolloService.js:1]: Please, rename `AppolloService` to just `client.js` and put it in some `util` folder. It is not a folder

File: src/services/ApolloService/index.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import { client } from './ApolloService';
```
Comment (Alan18081) [inline src/services/ApolloService/index.js:1]: We don't need this file

Comment (ecl1pseo) [reply src/services/ApolloService/index.js]: deleted

File: src/services/ApolloService/ApolloService.js (hunk 3 above)
Comment (ecl1pseo) [reply src/services/ApolloService/ApolloService.js]: Renamed and moved to utils folder

**Pull request merged**